source/community.applications/usr/local/emhttp/plugins/community.applications/include/previous_apps_helpers.php

```
@@ -0,0 +1,431 @@
+
+			$displayed[] = $template;
+		}
+
+		$displayed = array_merge($displayed, self::collectInstalledLanguagePacks($templates, $isActionCentre, $updateCount));
+
+		return $displayed;
+	}
+
+	private static function collectInstalledLanguagePacks($templates, $isActionCentre, &$updateCount) {
+		$displayed = [];
+		$languagesDir = CA_PATHS['languageInstalled'] ?? null;
```

PHPStan (2.1.51)

[error] Constant CA_PATHS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)